docs-client/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+const defaultConfig = require('../settings/eslint/eslintrc');
+
+// Ignore 'webpack.config.ts' so that it doesn't complain that
+// some dependencies must be moved from devDependencies to dependencies.
```

Comment:
We could also just move everything into `dependencies`. For apps not published to NPM it's one reasonable convention, which is followed by `create-react-app`

Reply:
That would be convenient, but I'm not sure if it's a good idea to move webpack and its plugins to `dependencies` because they are used only when building obviously.

Reply:
Technically everything is only when building since we build a bundle rather than publishing anything - I think that's why create-react-app follows that convention. But it's ok if it's easy to ignore files too

Reply:
Merged into `dependencies` for simplicity